examples/gauge-metrics/gauge_metrics.server.out

```diff
@@ -1,9 +1,9 @@
 # To start the service, navigate to the directory that contains the
-# `.bal` file and execute the `bal run` command below with the build time flag `--observability-included` and the runtime flag `--b7a.observability.metrics.enabled=true`.
-bal run --observability-included gauge_metrics.bal --b7a.observability.metrics.enabled=true
+# `.bal` file and execute the `bal run` command below with the `--observability-included` build time flag and the `Config.toml` runtime configuration file.
+BALCONFIGFILE=Config.toml bal run --observability-included gauge_metrics.bal
 
-ballerina: enabled prometheus metrics reporter
 [ballerina/http] started HTTP/WS listener 0.0.0.0:9797
+ballerina: enabled prometheus metrics reporter
```

Comment:
This should be updated to the latest changes in Prometheus

Reply:
Fixed with 6469f7b3e41b70893af628ad972f36ae59abad70